### DISCUSSION TITLE
2022-08-03 Weekly Meeting 67 Agenda - Content Portability

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#263] [Weekly Meetings]
Weekly Meeting 67 is happening on Wednesday August 3th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Check in on MSF

### Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

### Discussions: 
- Content Portability
![ContentInterop-080322](https://user-images.githubusercontent.com/63426722/182655826-93bde434-9519-4e20-8176-a4d7a7154503.png)

### Future: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Avatars` Pt II
- `Portals` Pt II 
- `Generative and AI tooling`

**Comment by mrmetaverse** [comment]
- new readme! 
- #265 
- backlog refinement: https://github.com/omigroup/omigroup/discussions/260#discussioncomment-3311330
- #260 
- we got into vket :small_airplane: LIke last time #89 
- Saracen has an ask that we should all take seriously.

**Comment by mrmetaverse** [comment]
# Content Portability Demo 

Sign up in a comment below :point_down:

**Comment by robertlong** [comment]
The OMI Scripting Group is proposing two things:
https://github.com/omigroup/omi-scripting-group/discussions/36

1. Moving the meetings to 1700 UTC on Thursdays. Adam often has conflicts at the current time and it's still a little early for those of us in the pacific time zone
2. Splitting the developer hour out of the working group so that it's clearer that it's not a formal working group but a casual time to talk about developer-related topics. Unsure how we want to track this on GitHub.

We need to hear from those people who are interested by next week. Post in the GH Discussion above with your thoughts.

**Comment by mrmetaverse** [comment]
Proposed a learn-while-we-build approach to world building. Can we build an OMI world and invite others to learn how?

**Reply by mrmetaverse** [reply]
similar to #217

**Comment by mrmetaverse** [comment]
# MSF-delegates 
- identifying top interest areas 
- on the 24th the organizers will analyze the top priorities 
- there is a large spreadsheet to add ideas to, message @humbletim if you'd like the link 
- other notes from recent meetings: https://github.com/omigroup/omigroup/discussions/265

**Reply by mrmetaverse** [reply]
OMI was mentioned today

**Reply by mrmetaverse** [reply]
One thing that was called out today is if you have a company and want to talk about it in the context of the Metaverse Standards forum, a lot of people have been stumped in describing what the MSF actually is. There are slides in the work for providing context about it without the fear of misrepresenting* MSF.

**Comment by mrmetaverse** [comment]
# Champions 
- Met Friday and Tuesday 
- We believe that the #188 Interoperator effort is satisfied by champions and groups like the MSF-delegates. 
- Refined the backlog and repos overall 
- archived inactive groups

**Comment by mrmetaverse** [comment]
# VR Chat 
- _"VRChat banned all user-made mods with just one day's notice, leaving many players feeling angry and betrayed"_ [source](https://kotaku.com/vrchat-virtual-reality-quest-anti-cheat-mods-facebook-1849341196)
- active protest from the community right now 
- corporate squeezing of a creator community, an exemplar of the state of affairs across a variety of platforms. 
- how we respond and provide space for creators is valuable
- can we provide any tools to help creators looking to get away and into a more inclusive and open place of the Metaverse? 
- if you are blogging, we have a blog and can syndicate the content to others as well. 
- Lets be mindful in how we engage, and how we ask questions of the broader community. 
- By being the ones to ask important questions we are modeling collaboration and inclusion.

**Comment by mrmetaverse** [comment]
# gLTF 
- there is a final spec of KHR Audio spec 
- still some work to do to be able to use it 
- instead of a demo day can we **have a hackathon to get these specs into a more artist-friendly workflow?**
  - need concentrated effort to push the tooling over the hump
- had a lot of churn in the spec but it is stabilized now. 
- discussed gLTF behavior graphs proposal 
- presenting a demo at SIGGRAPH tomorrow as well, appears to be going forward within Khronos group. Looking to identify how we can get involved as OMI should anyone want to participate in the spec (without needing a full membership in Khronos) 
- we will add link to google slides <-

**Reply by mrmetaverse** [reply]
MSF refers to these as plug-fests, the development of plugins to accelerate standards.

**Comment by mrmetaverse** [comment]
# Media 
- going to coordinate with blog post re: VR chat mentioned above :point_up: 
- if you have any content to be worked on, we can help 
- we are having major issues producing events with audio on discord. Bot has failed, and we don't know why. Audio was not captured during last weeks demo #258 :disappointed: 
- looking to re-record individuals who demoed last week to re-release the content and ensure your demo is heard. 
- In future events we will try to stream to discord and not out of discord. 
- Big thanks to the community for tweeting and using the hashtags. We have a system where if you tweet from your account and tag `@open_metaverse` and use hashtags like `OMI` `interoperability` `metaverse` it syncs with discord.

**Comment by mrmetaverse** [comment]
Proposed rethinking the gLTF group name to something bigger. Model interoperability for example. A group to address other formats like USD.
